src/i18n/locales/en-US.json

```diff
@@ -136,14 +136,14 @@
   "components.settings.changewalletname.changeButton": "Change name",
   "components.settings.changewalletname.title": "Change wallet name",
   "components.settings.changewalletname.walletNameInputLabel": "Wallet name",
-  "components.settings.removewalletscreen.descriptionParagraph1": "If you really wish to permanently delete the wallet make sure you have written down your 15-word recovery phrase.",
-  "components.settings.removewalletscreen.descriptionParagraph2": "To confirm this operation type the wallet name below.",
-  "components.settings.removewalletscreen.hasWrittenDownMnemonic": "I have written down mnemonic of this wallet and understand that I cannot recover the wallet without it.",
```

Comment:
Avoid the word "mnemonic" since we usually refer to this as the "recovery phrase"